core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -401,6 +402,9 @@ export class LoggedInUser extends BaseUser {
     await this.page.$eval(weCannotContentId, element =>
       element.getElementsByTagName('a')[0].click()
     );
```

Comment:
This test is intermittently failing due to a race condition between two navigations. After clicking the '420 million' link, the test is not waiting for the resulting navigation to complete before the beforeEach block started a new navigation. This lefts the first navigation unresolved, which sometimes leads to failures.
 Despite the test logging the final showMessage, it is not properly handling the navigation it initiated.

Reply:
Great find, @Akhilesh-max!